zephyr/module.yml

```diff
@@ -1,5 +1,10 @@
+# Copyright (c) 2022 Legrand North America, LLC.
```

Comment:
If you are gong to add copyright list here, please add Nordic semiconductor to it as well.
`# Copyright (c) 2019-2021 Nordic Semiconductor ASA`

Reply:
Added.